index.ts

```diff
@@ -20,13 +20,17 @@ export interface FeatureData {
     usage_stats?: usage_stats_url | [usage_stats_url, usage_stats_url, ...usage_stats_url[]];  // A single URL or an array of two or more
 }
 
-type browserIdentifier = "chrome" | "edge" | "firefox" | "safari";
+type browserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
 
 interface SupportStatus {
-    /** Whether the feature achieved baseline status */
+    /** Whether the feature achieved Baseline status */
     is_baseline: boolean;
-    /** Date the feature achieved baseline status */
+    /** Whether the feature is widely supported (high) or newly supported (low) */
+    substatus?: "high" | "low";
```

Comment:
`substatus` should be required if `is_baseline` is true, and forbidden otherwise. Rather than expressing this in the schema, how about simplifying to an enum of possible states, like this: `baseline: false | "high" | "low"`

A mix of booleans and strings is similar to BCD's `version_added`, and means it's possible to still use `baseline` like a boolean.

Reply:
OK, I've switched to a `baseline` field with one of the strings or `false`.